### DISCUSSION TITLE
一个解决pdf翻译排版问题导致字符覆盖显示的办法

### DISCUSSION THREAD

**Discussion by ygm-github** [immersive-translate/immersive-translate#592] [Ideas]
### 新功能 | New Feature

1. 对于pdf翻译排版问题导致的字符覆盖显示，我认为这是原PDF语言与翻译后的语言在字体大小以及字体数量上存在区别导致的。
2. 翻译后的PDF虽然和原文件整体排版一致，但对翻译后每一段内容的长度并没有限制，这就导致了字符覆盖重叠显示的问题。
3. 根据PDF原文档前一段与后一段的距离可确定翻译后每一段文档原本的长度
4. 我尝试手动对翻译后每一段要显示的内容增加了一个高度限制，如果超出指定长度则添加滚动条，似乎效果不错：
（以第一段为例，max-height: 26%; overflow: auto;）
![image](https://github.com/immersive-translate/immersive-translate/assets/80095326/895bddb5-1ffb-4034-b14a-1c32d2659fb1)

未修改前的效果如下：
![image](https://github.com/immersive-translate/immersive-translate/assets/80095326/206e113d-865e-41a0-973c-dcb0233b9689)

### 为什么您想要添加这些功能 | Why you want to add these features

这个翻译软件真的很酷。

**Comment by theowenyoung** [comment]
似乎很有道理，我们试试～

**Comment by keyboard3** [comment]
0.6.13 已采纳你的建议，同时支持了段落的编辑能力
